spec/unit/commands/import_command/openapi/update_service_proxy_spec.rb

```diff
@@ -153,5 +155,31 @@
         subject
       end
     end
+
+    context 'oidc issuer type (default) set' do
+      let(:security) { { id: 'oidc', type: 'oauth2', flow: :implicit_flow_enabled } }
```

Comment:
I see that this `:security` needs to be there. It is, though, a bad smell (probably my fault defining default values in parent contexts). No need to change it, I just wanted to mention.